Automattic/wp-calypso
# Fixes a zoom issue in iOs for the Advertising page - search input

Related to SELFSERVE-573

It fixes a zoom issue when a user from an iOS device clicks the Search box on the Advertising page.
The problem is that Safari will zoom in whenever it encounters an input text with a font-size under 16px.

## Proposed Changes

* Added a support query only enabled for ios that will change the text-box font-size to 16px, only for iOs. That way we can prevent the zoom that Safari makes.

**Previous version:**
![Screenshot 2023-07-20 at 16 23 49](https://github.com/Automattic/wp-calypso/assets/1258162/c2808b15-489f-4656-8c5d-23db2162a394)

**After the change:**
![Screenshot 2023-07-20 at 16 24 11](https://github.com/Automattic/wp-calypso/assets/1258162/1f1f3bb2-d0db-416d-b617-eb16544bc9cf)

## Testing Instructions

You will need an iOS devise to test this PR. It can be tested using a Simulator (if you have a macOS).

* In your simulator, log in to your WordPress account
* After that, open the live preview pointing to the Advertising page, and append this to the URL to bypass the oauth flags: `?flags=oauth` (you can also use your local installation instead of the live preview)
* When the page loads, click on the search box in the posts and campaigns page. 
* Verify that the page didn't zoom in

You can check the current behavior by going to Wordpress.com and clicking the same textbox.

## Pre-merge Checklist
